devdocs/process.md

```diff
@@ -68,8 +66,9 @@ This workflow is an adaptation of the [GitHub flow](https://guides.github.com/in
    to your branch
         
        ```
-       git pull upstream 
-       git merge master
+       git fetch upstream 
+       git checkout master
+       git merge upstream/master
```

Comment:
Isn't this part of the instructions about merging master into the branch for the issue? In this context, it makes sense to <code>git merge master</code> to get the latest changes on your branch.